tests/acceptance/features/bootstrap/CliContext.php

```diff
@@ -118,6 +120,25 @@ public function theAdministratorRemovesAllVersionsOfResources() {
 		$this->featureContext->setResponse(CliHelper::runCommand($body));
 	}
 
+	/**
+	 * @When the administrator removes the versions of the file :file of user :user from space :space using the CLI
```

Comment:
just a little one. others are good
```suggestion
	 * @When the administrator removes the versions of file :file of user :user from space :space using the CLI
```